src/amp-story-player/amp-story-player-impl.js

```diff
@@ -1264,6 +1281,32 @@ export class AmpStoryPlayer {
     );
   }
 
+  /**
+   * Rewinds the given story.
+   * @param {string} storyUrl
+   */
+  rewind(storyUrl) {
+    const storyIdx = storyUrl
+      ? findIndex(this.stories_, ({href}) => href === storyUrl)
+      : this.currentIdx_;
+
+    const story = this.stories_[storyIdx];
+    if (!story) {
+      throw new Error(`Story URL not found in the player: ${storyUrl}`);
+    }
+
+    let connectedPromise = Promise.resolve();
+    if (!story.iframe.isConnected) {
+      connectedPromise = story.connectedDeferred.promise;
+    }
+
+    connectedPromise.then(() => {
+      story.messagingPromise.then((messaging) =>
+        messaging.sendRequest('selectPage', {'rewind': true})
```

Comment:
I think it'd make sense to create a new `rewind` or `replay` message, it does more than `selectPage`